src/documents/ComponentConfigurationOptions.md

```diff
@@ -112,40 +109,59 @@ The following options are supported by all component grades:
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="Invokers.md">Invokers</a></td>
+    <td><a href="InfusionEventSystem.md">Infusion Event System</a></td>
   </tr>
 </table>
 
-### members ###
+### `listeners` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object defining properties to be added to the component object. These can be anything, including methods, strings, objects, etc. Definitions are evaluated as IoC expressions.</td>
+    <td>A record defining listener functions for the events supported by a component.</td>
   </tr>
   <tr>
     <th>Notes</th>
-    <td><code>members</code> differ from <code>invokers</code> in that the arguments of members are not resolved at invocation time.
-        The right-hand-side may contain an <a href="ExpansionOfComponentOptions.md">expander</a> definition, which may perhaps itself resolve onto an <a href="Invokers.md">invoker</a>.</td>
+    <td>Both component developers and integrators can define listeners for events.
+<a href="Invokers.md">Invokers</a> and <a href="ExpansionOfComponentOptions.md">Expanders</a> can be used as listeners here. Note that as well as being a simple string holding the name of an event on this component, a listener key may also be a full <a href="IoCReferences.md">IoC Reference</a> to any other event held in the component tree (for example <code>"{parentComponent}.events.parentEvent"</code>. As well as being a simple function name, a the value associated with the key may be a <a href="InfusionEventSystem.md">Listener Record</a> or else follow the syntax of an invoker indicating that the registered listener receives a different signature from the one that the event has fired (see <a href="EventInjectionAndBoiling.md">Event injection and boiling</a>).</td>
```

Comment:
Is it true that an expander "can be used as a listener here," or is it slightly more subtle (i.e. the result of an expander can be used as a listener)?